src/spatialdata/models/models.py

```diff
@@ -792,3 +869,59 @@ def _validate_and_return(
     if isinstance(e, AnnData):
         return _validate_and_return(TableModel, e)
     raise TypeError(f"Unsupported type {type(e)}")
+
+
+def get_table_keys(table: AnnData) -> tuple[str | list[str], str, str]:
+    """
+    Get the table keys giving information about what spatial element is annotated.
+
+    The first element returned gives information regarding which spatial elements are annotated by the table, the second
+    element gives information which column in table.obs contains the information which spatial element is annotated
+    by each row in the table and the instance key indicates the column in obs giving information of the id of each row.
+
+    Parameters
+    ----------
+    table:
+        AnnData table for which to retrieve the spatialdata_attrs keys.
+
+    Returns
+    -------
+    None | tuple[str | list[str], str, str]
+        The keys in table.uns['spatialdata_attrs']
+    """
+    if table.uns.get(TableModel.ATTRS_KEY):
+        attrs = table.uns[TableModel.ATTRS_KEY]
+        return attrs[TableModel.REGION_KEY], attrs[TableModel.REGION_KEY_KEY], attrs[TableModel.INSTANCE_KEY]
+
+    raise ValueError("No spatialdata_attrs key found in table.uns. Therefore, no table keys found.")
+
+
+def check_target_region_column_symmetry(table: AnnData, region_key: str, target: str | pd.Series) -> None:
```

Comment:
Useful function. I would add an example in the docstring since I think it would not be immediatly clear to the user when to use it.

Reply:
Resolved with [6123f9d](https://github.com/scverse/spatialdata/pull/410/commits/6123f9d03c7ccc839d4a7950d75f46e7626aebff)